playbooks/roles/postgresql_aggregation/defaults/main.yml

```diff
@@ -23,3 +23,12 @@ POSTGRESQL_AGGREGATION_PASSWORD: "password"
 POSTGRESQL_AGGREGATION_USER_ROLE: "boxes"
 POSTGRESQL_AGGREGATION_PORT: 5432
 POSTGRESQL_AGGREGATION_OPTIONS: {}
+
+postgresql_aggregation_client_cert: ''
+postgresql_aggregation_client_key: ''
+postgresql_aggregation_ca_cert: ''
+postgresql_aggregation_client_cert_path: '/etc/ssl/certs/postgresql_aggregation-client-cert.pem'
+postgresql_aggregation_client_key_path: '/etc/ssl/certs/postgresql_aggregation-client-key.pem'
+postgresql_aggregation_ca_cert_path: '/etc/ssl/certs/postgresql_aggregation-server-ca.pem'
+postgresql_aggregation_cert_owner: "{{ edxapp_user }}"
+postgresql_aggregation_cert_group: "{{ edxapp_user }}"
```

Comment:
@tkeemon @bryanlandia Thanks for the review. I agree with you, the problem is that PostgreSQL has some rules for the key perms, they can't have any access for world or group, and since the `edxapp` is the one that is going to use it, I decided to set it as owner.

Please take a look at the PostgreSQL docs: https://www.postgresql.org/docs/9.1/static/ssl-tcp.html

> On Unix systems, the permissions on server.key must disallow any access to world or group; achieve this by the command chmod 0600 server.key